ubuntu_bug_triage/__main__.py

```diff
@@ -73,6 +73,11 @@ def parse_args():
         + ", ".join(ACTIONABLE_BUG_STATUSES)
         + ".",
     )
+    parser.add_argument(
+        "--ignore",
+        nargs='*',
+        help="""ignore bugs edited last by the listed person""",
+    )
```

Comment:
1) How about `--ignore-user`? I'm not set on that, but `--ignore` as you said is too generic. There is clearly a use case for `--ignore-project` as well down the road as right now I have the hardcoded blacklist in `__init__.py`.

2) Set `default=[]` and we can remove your extra check below.

3) Can you move this up after --debug and before --json. It is not entirely obvious, but they are in alphabetical order per the naming of the `--<option>` value. Except for `--include-project` which I just noticed... oops

Reply:
I've done this changes now